configs/migrations/redshift/20211215141537_create_incidents_view.up.sql

```diff
@@ -16,6 +16,9 @@ CREATE OR REPLACE VIEW watchops.incidents AS (
             WHEN source LIKE 'gitlab%' THEN TO_TIMESTAMP(json_extract_path_text(json_serialize(metadata), 'object_attributes', 'closed_at'), 'YYYY-MM-DD HH24:MI:SS') 
             WHEN source LIKE 'opsgenie' THEN dateadd(ms, CAST(json_extract_path_text(json_extract_path_text(json_serialize(metadata), 'alert'), 'updatedAt') AS bigint), '1970-01-01')
         end as time_resolved,
+        CASE WHEN source LIKE 'opsgenie' THEN regexp_substr(json_extract_path_text(json_serialize(metadata), 'alert', 'note'),'[^:\s]*$') 
```

Comment:
I think we can apply this to any source and not only to opsgenie. All sources will need to have the root cause in it. And can regex it from the metadata field directly.

Reply:
you're right! I added it in the new commit